activate.js

```diff
@@ -0,0 +1,36 @@
+var carLot = (function (carStuff) {
+//calls all functions when box is clicked
+  carStuff.beginEvents = function (event) {
```

Comment:
In requirement 3 it specifies that the function should be named `activateEvents`. I know that seems super picky, but if you're working in a team or on old code you might have to name things to match existing code or to fit in properly with other people's code.